Anixe.Ion/Anixe.Ion.csproj

```diff
@@ -3,7 +3,9 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.0</TargetFramework>

```

Comment:
will it be better to move it to netstandard2.0 ?

Reply:
IonReader uses ArrayPool from System.Buffers that does not exists in netstandard2.0

Reply:
it can be compiled with:
```xml
<TargetFrameworks>netcoreapp2.0;netstandard2.1</TargetFrameworks>
```
should I do this change?

Reply:
@wmaryszczak ?